src/content/learn/importing-and-exporting-components.md

```diff
@@ -282,7 +283,7 @@ export default function App() {
 ```
 
 ```js Gallery.js active
-// Move me to Profile.js!
+// Pindahkan aku ke Profile.js!
```

Comment:
Because we use `Anda` to translate `you`, therefore shouldn't we use a similar formality with `saya`?

```suggestion
// Pindahkan saya ke Profile.js!
```

More discussion: https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5745464

Reply:
@aroyan @resir014 Please clarify which one to use.

Reply:
> @aroyan @resir014 Please clarify which one to use.

I think it's better to use `Para Ilmuwan`